src/doc/src/reference/unstable.md

```diff
@@ -296,7 +296,21 @@ The 'public-dependency' feature allows marking dependencies as 'public'
 or 'private'. When this feature is enabled, additional information is passed to rustc to allow
 the 'exported_private_dependencies' lint to function properly.
 
-This requires the appropriate key to be set in `cargo-features`:
+To enable this feature, you can either use `-Zpublic-dependency`
+
+```sh
+cargo +nightly run -Zpublic-dependency
+```
+
+or `[unstable]` table, for example,
+
+```toml
+# .cargo/config.toml
+[unstable]
+public-dependency = true
+```
+
+Another way not recommended is set the appropriate key in `cargo-features`,
 
 ```toml
 cargo-features = ["public-dependency"]
```

Comment:
This is going to be phased out, right? Should we either remove this, or make it clear on that deprecation plan? Like

You could also

```suggestion
`public-dependency` could also be enabled in `cargo-features`, though this is deprecated and will be removed soon.

```toml
# in Cargo.toml
cargo-features = ["public-dependency"]
```

Reply:
Yeah, I was gonna write that it's gonna be deprecated soon. 

It is recommended that you update the documentation when you remove the code.